InfiniteLoopLMSSQL/src/com/st/lmssql/utils/DateCalculations.java

```diff
@@ -0,0 +1,61 @@
+package com.st.lmssql.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class DateCalculations {
+
+	//gets today's date in yyyy-MM-dd
+	public static Date getCurrentTime() {
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
+		String currentTime = sdf.format(new Date());
+		
+		Date utilDate = null;
+		try {
+			utilDate = sdf.parse(currentTime); //in utilDate

```

Comment:
Here you create a `Date`, format it as a `String`, and then immediately parse that formatted `String` to get another `Date`. This seems unnecessary.